pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -386,3 +386,49 @@ func createEcDoubleSlopeV1Hinter(resolver hintReferenceResolver) (hinter.Hinter,
 
 	return newEcDoubleSlopeV1Hint(point), nil
 }
+
+func newReduceV1Hinter(x hinter.ResOperander) hinter.Hinter{
+	return &GenericZeroHinter{
+		Name:"reduceV1",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
+			//> x = pack(ids.x, PRIME) % SECP_P
+
+			secPBig, ok := secp_utils.GetSecPBig()
+			if !ok{
+				return fmt.Errorf("GetSecPBig failed")
+			}
+			xAddr, err := x.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			xMemoryValues, err := hinter.GetConsecutiveValues(vm, xAddr, int16(3))
+            if err != nil {
+                return err
+            }
+			var xValues [3]*fp.Element
+            for i := 0; i < 3; i++ {
+                xValue, err := xMemoryValues[i].FieldElement()
+                if err != nil {
+                    return err
+                }
+                xValues[i] = xValue
+            }
+			xBig, err := secp_utils.SecPPacked(xValues)
+            if err != nil {
+                return err
+            }
+			xBig.Mod(&xBig, &secPBig)
+			return ctx.ScopeManager.AssignVariable("x", xBig)
```

Comment:
Just noticed that I asked to assign it to `x` instead of `value`. But turns out the code comment in the function is wrong! 

Please update `//> x = pack(ids.x, PRIME) % SECP_P` to `//> value = pack(ids.x, PRIME) % SECP_P`